doc/ranger.pod

```diff
@@ -887,12 +887,16 @@ if you delete multiple files at once.
 
 The date format to be displayed in the interface. It defaults to "%x" which is the locale
 appropriate date format. See python's I<strftime> docs for valid format strings.
+To avoid ranger trying to resolve the format strings as macros, they need to be
```

Comment:
Maybe we should merge these sections to avoid duplication? Like I did for the map variations. Also, maybe a more thorough explanation's in order? Explain that doubling up the `%%`'s is a good habit because *some* formatting characters need the escaping, like `%d`.

Reply:
Do you mean adding a new topic under the *CONCEPTS* section in **doc/ranger.pod**?

Reply:
No, I meant empty sections so you only have to do the explanation once. Look at `map/cmap/tmap/...` to see what I mean. The rendered result puts the headers together so it looks like the explanation applies to all the commands or settings in this case.